User.ts

```diff
@@ -20,19 +19,13 @@ export default class User implements IUser {
     /*
      * Loads the roles of the user
      */
-    async loadRoles() {
+    async loadRoles() : Promise<void> {
 
         // TODO: Implement logic on SC-server instead of webdav with a specific flag
 
-        const res = await fetch(environment.BASE_URL + '/roles/user/' + this.uid, {
-            headers: {
-                'Authorization': 'Bearer ' + this.jwt
-            }
-        })
-
-        const data = await res.json()
+        const res = await api({jwt : this.jwt}).get('/roles/user/' + this.uid);
```

Comment:
Do we really need the `JWT` as a key only for this special case? Wouldn't it work by calling `api({user: this})` and therefore remove the extra property `JWT`?